prime-router/src/main/kotlin/history/db/ReportGraph.kt

```diff
@@ -0,0 +1,145 @@
+package gov.cdc.prime.router.history.db
+
+import gov.cdc.prime.router.azure.DatabaseAccess
+import gov.cdc.prime.router.azure.db.Tables.COVID_RESULT_METADATA
+import gov.cdc.prime.router.azure.db.Tables.REPORT_FILE
+import gov.cdc.prime.router.azure.db.Tables.REPORT_LINEAGE
+import gov.cdc.prime.router.azure.db.tables.pojos.CovidResultMetadata
+import gov.cdc.prime.router.common.BaseEngine
+import org.apache.logging.log4j.kotlin.Logging
+import org.jooq.CommonTableExpression
+import org.jooq.Record
+import org.jooq.Record2
+import org.jooq.impl.DSL
+import org.jooq.impl.DSL.selectDistinct
+import org.jooq.impl.SQLDataType
+import java.util.UUID
+
+private const val PARENT_REPORT_ID_FIELD = "parent_report_id"
+
+private const val CHILD_REPORT_ID_FIELD = "child_report_id"
+
+private const val METADATA_CTE = "metadata"
+
+private const val PATH_FIELD = "path"
+
+private const val SOURCE_CTE = "source"
+
+private const val LINEAGE_CTE = "lineage"
```

Comment:
The way we have our usage of these CTEs designed, these names will never be referenced outside of their own recursive definitions? If so, do we really need to break these out into class level consts?

Reply:
Yea the actual CTE names never referenced, happy to pull the string back into the function.  The paths though do get used in a few places.